pkg/types/azure/defaults/machines.go

```diff
@@ -0,0 +1,20 @@
+package defaults
+
+import (
+	"fmt"
+)
+
+func BootstrapMachineType(region string) string {
+	instanceClass := InstanceClass(region)
+	return fmt.Sprintf("%s_D4s_v3", instanceClass)
```

Comment:
this should be `_DS4_v2`

Reply:
looks like v3 are named correctly like https://github.com/Azure/azure-sdk-for-go/blob/784a56e5c68f892e337b82c8e1feb5fb6cd7a58a/services/containerservice/mgmt/2018-03-31/containerservice/models.go#L197